### ISSUE TITLE
RPC sendtransaction destination empty

### ISSUE THREAD

**Issue by streamcoindev** [cryptonotefoundation/cryptonote#158]
im getting destination empty when trying to send coins im working on a web wallet already have wallet generation and balance checker however when im trying to send coins i get destination empty can not find anywhere what this exactly means

**Comment by SoraKohaku** [comment]
json old problem used new updates~

**Comment by streamcoindev** [comment]
So you say rebuild the coin and it should begood ?

**Comment by SoraKohaku** [comment]
yes i think so. but i already upgraded code half work for boost 1.6x
